### ISSUE TITLE
from library with kids movies: 54 movies. 4 no info

### ISSUE THREAD

**Issue by michpas** [ilarramendi/Cover-Ratings#33]
In my kids library I have 54 movies. 51 processed correctly.
For 4 there is no .png post or thumb file created.

What can we do to understand why?

**Comment by michpas** [comment]
I found the issue.
On the first run, these folders all had (for some reason) the movie in the folder twice.
eg. moviename.mkv and moviename2.mkv. Same size, same movie; probably a dl error.

This explains the errors.
In the first run, this created the errors that I saw in teh log.
I than removed the duplicated movies.
The errors dissapeared from the next run/log, but there would still not be an new image created.
It was only when I cleared all config files (eg database?) and did a fresh run, all were created fine.

So there is definately something going wrong with 2 movies with roughly same titles and size in one folder and even if that is fixed, the script doesn't recover from it; 
Not sure how to best solve, as in practise there should only be 1 movie in each folder;
